### PR TITLE
[Tools] Enable PHPCS for data_integrity/score_instrument.php

### PR DESCRIPTION
**Pull request by laemtl**

### CODE REVIEW

File: tools/data_integrity/score_instrument.php [hunk 1]
```diff
@@ -38,33 +38,47 @@
...
+    && $argv[2] === 'all'
+    && isset($argv[3])
+    && $argv[3] !== 'inprogress')
+    || ($argv[1] !== 'all'
+    && $argv[2] === 'one'
+    && isset($argv[5])
+    && $argv[5] !== 'inprogress')
 ) {
-    echo "\nERROR: Too many or incorrect arguments passed. Check documentation and try again.";
+    echo "\n
```
Comment (driusan) [inline tools/data_integrity/score_instrument.php:70]: This changes the value. The string sent to the terminal was:

```
ERROR: Too many or incorrect arguments passed. Check documentation and try again.
```

now it's

```

Error: Too many or or incorrect arguments passed.
Check documentation and try again.
```

with an extra newline at the start and a new one in the middle of the message.

File: tools/data_integrity/score_instrument.php [hunk 2]
```diff
@@ -221,39 +261,55 @@ function logMessage($message)
...
+ * @return void
+ */
 function showHelp()
 {
     echo "\n\n*** Score Instruments ***\n\n";
 
     echo "Usage:
-    score_instrument.php [help | -h]                                       -> displays this message
+    score_instrument.php [help | -h]                                       
+    -> displays this message
```
Comment (driusan) [inline tools/data_integrity/score_instrument.php:303]: This is also changing the help output and losing the formatting.

**Comment by laemtl** [comment]
@driusan Solved!

**Review by pierre-p-s — APPROVED** [review]

**Comment by driusan** [comment]
Looks fine, but there are conflicts

**Comment by laemtl** [comment]
@driusan Ready.

**Comment by laemtl** [comment]
@driusan Rebased and ready for merging.

**Comment by laemtl** [comment]
@driusan Rebased and ready for merging.

**Review by driusan — APPROVED** [review]
Seems to be entirely whitespace/array syntax changes.

**Pull request merged**